### ISSUE TITLE
> We trained for around 8 hours for SegViT Large for the Pascal-context dataset with a node of 4 A100.

### ISSUE THREAD

**Issue by 1787648106** [zbwxp/SegVit#9]
> We trained for around 8 hours for SegViT Large for the Pascal-context dataset with a node of 4 A100.

Hi, I have a naive question, is the pacscal context configuration defaulted to 60 categories? If I want to change the pascal context to include 59 classes, it is model=[...decode_head=dict(...num_classes=59...) in configs/segvit/segvit...480..80k_pc.py ..] and loss_decode=dict(...num_classes=59...)? Thanks again!
![QQ图片20230513144906](https://github.com/zbwxp/SegVit/assets/22196901/0e2e9f90-b2a1-4f12-8ef3-7214ad929308)

_Originally posted by @1787648106 in https://github.com/zbwxp/SegVit/issues/7#issuecomment-1546548300_

**Comment by AkideLiu** [comment]
It is advisable to consider adjusting the number of classes in the model section. Furthermore, the dataset loading section should be modified according to the settings in mmseg. For detailed guidance on this matter, please refer to the following link: 

https://github.com/open-mmlab/mmsegmentation/blob/0.x/configs/_base_/datasets/pascal_context_59.py

**Comment by 1787648106** [comment]
> adjusting the number of classes in the model sectio

Thank you for your reply, have a nice day

**Comment by 1787648106** [comment]
> It is advisable to consider adjusting the number of classes in the model section. Furthermore, the dataset loading section should be modified according to the settings in mmseg. For detailed guidance on this matter, please refer to the following link:
> 
> https://github.com/open-mmlab/mmsegmentation/blob/0.x/configs/_base_/datasets/pascal_context_59.py

When I followed your guide to modify the relevant files and changed the 60 category to 59 category for training, I found that the miou will be very low when the training is about to be completed, about 0.18. The picture below shows all my modifications. Is there something wrong with the settings?
![image](https://github.com/zbwxp/SegVit/assets/22196901/e6700517-dba6-41a7-9d8f-06cdc9fa803f)
![image](https://github.com/zbwxp/SegVit/assets/22196901/c49a2c39-7ca5-4ef7-b01c-494c0b3fc945)
In addition, my experimental settings are consistent with those in the paper, the batch size is 16, the hardware is 4 A800 graphics cards, and 2T of ram memory.
